programs/vote/src/vote_state.rs

```diff
@@ -423,6 +452,17 @@ pub fn process_vote(
     verify_authorized_signer(&vote_state.authorized_voter, signers)?;
 
     vote_state.process_vote(vote, slot_hashes, clock.epoch)?;
+    if let Some(timestamp) = vote.timestamp {
+        let slot = vote
+            .slots
+            .iter()
+            .max()
+            .expect("vote must contain at least one slot");
```

Comment:
you mustn't expect() in these programs, you must return an error....

Reply:
Duh, forgot where I was. Thanks!